addrmgr/addrmanager.go

```diff
@@ -55,6 +55,8 @@ type serializedKnownAddress struct {
 	TimeStamp   int64
 	LastAttempt int64
 	LastSuccess int64
+	Services    wire.ServiceFlag
```

Comment:
Don't see these new fields being set in `savePeers`. Also it'd be nice to get some basic test coverage for this new addition to ensure that serialization version bump worked properly.